### DISCUSSION TITLE
json road 에러

### DISCUSSION THREAD

**Discussion by popcorn-kim93** [dsdanielpark/Bard-API#276] [Q&A]
안녕하세요! bard api 를 사용중인 사람입니다!

최근 잼미니(?) 업데이트로 응답 형식이 조금 변경된것같은데 코드에 있는 많은 매직넘버들 중에 에러가 발생한듯합니다...

```
in AskChatbot
    response = ChatBards[uid].bard.get_answer(ask)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/bardapi/core.py", line 443, in get_answer
    resp_dict = json.loads(resp.content.splitlines()[-5])[0][2]
                           ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^
```

위는 에러 코드입니다..

**Comment by dsdanielpark** [comment]
안녕하세요!

저는 snim0e부터 접근이 안되서 디버깅을 못 하고 있습니다. 해당 에러에 대한 디버깅은 어렵지 않은 것 같은데, 현재 단일 쿠키로도
정상적으로 작동하는지 궁금합니다.
한국에서 사용 중이시라면 지역보다는 계정 문제일 가능성이 높겠군요.

소중한 피드백 감사드립니다.


2024년 2월 9일 (금) 오후 12:02, 강냉이 ***@***.***>님이 작성:
